test/testinput/bufr_gpsipw.yaml

```diff
@@ -0,0 +1,82 @@
+# (C) Copyright 2022 NOAA/NWS/NCEP/EMC
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+
+observations:
+  - obs space:
+      name: bufr
+      obsdatain: "./testinput/gdas.20240806t00z.gpsipw.tm00.bufr_d.nr"
+
+      exports:
+        variables:
+          # MetaData
+          timestamp:
+            datetime:
+              year: "*/YEAR"
+              month: "*/MNTH"
+              day: "*/DAYS"
+              hour: "*/HOUR"
+              minute: "*/MINU"
+
+          latitude:
+            query: "*/CLATH"
+
+          longitude:
+            query: "*/CLONH"
+
+          stationName:
+            query: "*/STSN"
+
+          heightOfStation:
+            query: "*/GNSSRPSQ/ELEV"
+
+          # ObsValue
+          precipitableWater:
+            query: "*/TPWT"
+
+    ioda:
+      backend: netcdf
+      obsdataout: "./testrun/20240806T00Z_ipw_gnssgb_ncep.nc"
+
+      globals:
+        - name: "platformCommonName"
+          type: string
```

Comment:
@mrixlam ,,, hearing from @gthompsnJCSDA that adding a line like the but `type: float` to the precipitableWater should force float

Reply:
Hi @BenjaminRuston we previously tried "type: float" for the variables section but it didnt work. But you are right. When I removed "type: float" from the variables section to the exports section, it works. The converted file now have float values for precipitableWater.

Reply:
I'm confused @mrixlam I just tried this with and w/o the `type: float`

when I added it, I got floats it worked correctly:
`range(/ObsValue/precipitableWater): [2.299999952316284, 72.0999984741211]`

when removed it output the integer values:
range(/ObsValue/precipitableWater): [2, 72]

so going to add it in... please confirm